src/api/rpc/full_node/index.ts

```diff
@@ -503,6 +503,162 @@ export async function get_fee_estimate<T extends TRPCAgent | TDaemon>(agent: T,
   return agent.sendMessage<R>(chia_full_node_service, get_fee_estimate_command, data);
 }
 
+
+
+// simulator RPCs
+export const get_all_blocks_command = "get_all_blocks";
+export type get_all_blocks_command = typeof get_all_blocks_command;
+export type TGetAllBlocksRequest = {
+};
+export type TGetAllBlocksResponse = {
+  blocks: FullBlock[];
+  success: bool;
+};
+export type WsGetAllBlocksMessage = GetMessageType<chia_full_node_service, get_all_blocks_command, TGetAllBlocksResponse>;
+export async function get_all_blocks<T extends TRPCAgent | TDaemon>(agent: T, data: TGetAllBlocksRequest){
+  type R = ResType<T, TGetAllBlocksResponse, WsGetAllBlocksMessage>;
+  return agent.sendMessage<R>(chia_full_node_service, get_all_blocks_command, data);
+}
+
+
+
+export const farm_block_command = "farm_block";
+export type farm_block_command = typeof farm_block_command;
+export type TFarmBlockRequest = {
+  address: str;
+  guarantee_tx_block?: bool;
+  blocks?: bool;
+};
+export type TFarmBlockResponse = {
+  new_peak_height: uint64;
```

Comment:
Regarding to the current implementation, the type of `new_peak_height` should be `int`.